public/editor/scripts/editor/js/fc/bramble-menus.js

```diff
@@ -79,6 +79,29 @@ define(function(require) {
       return false;
     });
 
+    //set the AutoComplete toggle to reflect whether auto-complete is enabled or disabled
+    if(bramble.getAutoComplete()) {
+        $("#autocomplete-toggle").addClass("switch-enabled");
+    } else {
+        $("#autocomplete-toggle").removeClass("switch-enabled");
+    }
+    // Enable/Disable Autocomplete
+    $("#autocomplete-toggle").click(function() {
+      // Toggle current value
+      var $autoCompleteToggle = $("#autocomplete-toggle");
+      var toggle = !($autoCompleteToggle.hasClass("switch-enabled"));
+
+      if(toggle) {
+        $autoCompleteToggle.addClass("switch-enabled");
+        bramble.stopAutoComplete();
```

Comment:
I asked for a naming change on these methods in the Brackets PR, since we tend to use `enable/disable` in the rest of the methods.